src/reaper_osara.cpp

```diff
@@ -1718,6 +1718,37 @@ void postSelectMultipleItems(int command) {
 	SetCursorContext(1, nullptr);
 }
 
+void postMoveItemOrEnvelopePoint(int command) {
+	TrackEnvelope* envelope = GetSelectedEnvelope(nullptr);
+	if (!envelope) {
+		int count = CountSelectedMediaItems(0);
+		if(count==0) {
+			outputMessage(translate("no items selected"));
+			return;
+		}
+		// Translators: Reported when moving items{} will be replaced by the
+	// number of items; e.g. "2 items moved".
```

Comment:
Unfortunately, you're going to need to put this translators comment (or a slight variation of it) above each message. Translators comments are only associated with the translatable string immediately following them. So, as this stands, the second message won't get a comment in the .po file.

```suggestion
		// Translators: Reported when moving items{} will be replaced by the
	// number of items; e.g. "2 items moved".
```

Reply:
```suggestion
```